### PR TITLE
feat(google-sheets): update and fix example

### PR DESCRIPTION
**Pull request by KyleAMathews**
Also removed most of the remaining usages of "starter" in repo + fixed a few typos.

### CODE REVIEW

File: examples/google-sheets/src/api/sheets.js [hunk 1]
```diff
@@ -9,12 +8,19 @@ const handler = async (req, res) => {
...
     const doc = new GoogleSpreadsheet(process.env.GOOGLE_SHEET_ID)
 
-    await doc.useServiceAccountAuth(creds)
+    await doc.useServiceAccountAuth({
+      client_email: process.env.GOOGLE_SERVICE_ACCOUNT_EMAIL,
+      private_key: process.env.GOOGLE_PRIVATE_KEY.replace(/\\n/gm, "\n"),
+    })
     await doc.getInfo()
 
-    const sheet = await doc.addSheet({
```
Comment (KyleAMathews) [inline examples/google-sheets/src/api/sheets.js:15]: this created a new sheet on every invocation

File: examples/google-sheets/src/pages/index.js [hunk 2]
```diff
@@ -17,28 +17,28 @@ const IndexPage = () => {
...
-            key={`row-${idx}`}
-            idx={idx}
-            state={rowState[idx]}
-            handleChange={handleChange}
-          />
-        ))}
-      </div>
-      <input type="submit" value="Submit" />
-    </form>
+    <div style={{ maxWidth: 1000 }}>
```
Comment (KyleAMathews) [inline examples/google-sheets/src/pages/index.js:21]: constrain the width

File: examples/google-sheets/src/api/sheets.js [hunk 3]
```diff
@@ -9,12 +8,19 @@ const handler = async (req, res) => {
 
     const doc = new GoogleSpreadsheet(process.env.GOOGLE_SHEET_ID)
 
-    await doc.useServiceAccountAuth(creds)
+    await doc.useServiceAccountAuth({
+      client_email: process.env.GOOGLE_SERVICE_ACCOUNT_EMAIL,
```
Comment (KyleAMathews) [inline examples/google-sheets/src/api/sheets.js:12]: use env variables — the JSON file would have to be committed to work on cloud

**Review by dmccraw — APPROVED** [review]
:ship: it!

**Pull request merged**